### ISSUE TITLE
Trouble installing on Ubuntu 21.04

### ISSUE THREAD

**Issue by ccaprani** [Cimbali/pympress#209]
### Environment
- Ubuntu 21.04
- Python version 3.8.11

### Pympress
- version ??
- Installation method: trying to do pip as apt version is behind.

### Expected behaviour
Had `apt-get ...` installed, but it is just v1.5.3 and I had Issue #188 and so would like to install latest (v.1.6.3)

### Steps to reproduce
Failing miserably. Git clone or download; navigate to folder, then `pip install .` fails with 

```python
    ERROR: Command errored out with exit status 1:
     command: /home/ccaprani/anaconda3/bin/python -c 'import io, os, sys, setuptools, tokenize; sys.argv[0] = '"'"'/tmp/pip-req-build-8ukncmwn/setup.py'"'"'; __file__='"'"'/tmp/pip-req-build-8ukncmwn/setup.py'"'"';f = getattr(tokenize, '"'"'open'"'"', open)(__file__) if os.path.exists(__file__) else io.StringIO('"'"'from setuptools import setup; setup()'"'"');code = f.read().replace('"'"'\r\n'"'"', '"'"'\n'"'"');f.close();exec(compile(code, __file__, '"'"'exec'"'"'))' egg_info --egg-base /tmp/pip-pip-egg-info-xkts644b
         cwd: /tmp/pip-req-build-8ukncmwn/
    Complete output (8 lines):
    Using setuptools.setup():
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "/tmp/pip-req-build-8ukncmwn/setup.py", line 369, in <module>
        setuptols_version = tuple(int(n) for n in setuptools.__version__.split('.'))
      File "/tmp/pip-req-build-8ukncmwn/setup.py", line 369, in <genexpr>
        setuptols_version = tuple(int(n) for n in setuptools.__version__.split('.'))
    ValueError: invalid literal for int() with base 10: 'post20210125'
    ----------------------------------------
WARNING: Discarding file:///home/ccaprani/Downloads/pympress-1.6.3. Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
ERROR: Command errored out with exit status 1: python setup.py egg_info Check the logs for full command output.
```

### Problem
I'd like to install it.

**Comment by Cimbali** [comment]
That fix is already in master. We parsed the `pip` version poorly, and a fix was contributed by @Mic92 in #208.

The temporary workaround until 1.6.4 is out is to install from master:
```
pip install git+https://github.com/Cimbali/pympress
```

**Comment by ccaprani** [comment]
You legend! Thank you!
